app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.startswith("cp "):
+        read_from, write_to = command[3:].split()
```

Comment:
Will be better to unpack it at the beginning of the function
command_cp, old_file, new_file = command.split()
to avoid usind slices and calling function startswith
then can just put `if command_cp == "cp" ` that's it